### PR TITLE
search as primary CTA JIRA ID CXPUI-4

### PR DESCRIPTION
**Pull request by sjyothi54**
OLD: Search bar was below the guided install tile
NEW:Moved the search bar into the header, below the ‘Our quickstarts bundle everything…’ text and also removed the Grid/List view button
Old view:
Screenshot 2022-02-11 at 5 09 56 PM
New view:
Screenshot 2022-02-11 at 5 09 05 PM

### CODE REVIEW

**Comment by sjyothi54** [comment]
JIRA: https://newrelic.atlassian.net/browse/CXPUI-13
Description:Demo Feedback-Search box design to be changed
Steps to testins: verified search box  UI with different resolutions
Old view:
Old_view


New View:
Screenshot 2022-02-16 at 4 17 30 PM
Screenshot 2022-02-17 at 2 17 32 PM

File: src/components/IOBanner.js [hunk 1]
```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import { css } from '@emotion/react';
 import bannerOverlayRight from '../images/io-banner/banner-style-right.svg';
 import bannerOverlayLeft from '../images/io-banner/banner-style-left.svg';
-
+import { SearchInput } from '@newrelic/gatsby-theme-newrelic';
+import { QUICKSTARTS_COLLAPSE_BREAKPOINT } from '../data/constants';
```
[the remark this answers is not in the document]
Comment (sjyothi54) [reply src/components/IOBanner.js:7]: I had done the necessary changes according to your requirements in file's css
Attached screenshots:
iphone 12 pro screen resolution:
Screenshot 2022-02-18 at 1 25 29 PM
iPad Mini:
Screenshot 2022-02-18 at 1 25 47 PM

**Comment by aswanson-nr** [comment]
Closing and reopening to trigger a build now that they are available

File: src/components/IOBanner.js [hunk 2]
```diff
@@ -2,10 +2,11 @@ import React from 'react';
...
 const MOBILE_BREAKPOINT = '530px';
 
-const BannerHeaderContent = () => (
+const BannerHeaderContent = ({ search, setSearch }) => (
```
[the remark this answers is not in the document]
Comment (sjyothi54) [reply src/components/IOBanner.js:9]: Sorry It was mistake, I added Prop types to IOBanner component

File: src/components/IOBanner.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { css } from '@emotion/react';
 import bannerOverlayRight from '../images/io-banner/banner-style-right.svg';
 import bannerOverlayLeft from '../images/io-banner/banner-style-left.svg';
-
+import { SearchInput } from '@newrelic/gatsby-theme-newrelic';
+import { QUICKSTARTS_COLLAPSE_BREAKPOINT } from '../data/constants';
 const MOBILE_BREAKPOINT = '530px';
```
[the remark this answers is not in the document]
Comment (sjyothi54) [reply src/components/IOBanner.js]: Yeah I had replaced QUICKSTARTS_COLLAPSE_BREAKPOINT in the place of MOBILE_BREAKPOINT

**Review by aswanson-nr — APPROVED** [review]
Looks good to go!

**Comment by aswanson-nr** [comment]
Hey @sjyothi54! This looks good to merge, it just has a merge conflict from the last PR that went in. Could you resolve that conflict? Then we can get this in!

**Comment by pkosanam** [comment]
Hi @aswanson-nr , I have resolved conflicts and updated this branch with newrelic:main 👍

**Review by aswanson-nr — APPROVED** [review]
Woohoo!

**Pull request merged**